### PR TITLE
[feature] Raw as a synonym for TSVRaw

### PR DESCRIPTION
**Pull request by Unalian**
### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Raw as a synonym for TSVRaw

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

<details>
    <summary>Modify your CI run</summary>

**NOTE:** If your merge the PR with modified CI you **MUST KNOW** what you are doing
**NOTE:** Checked options will be applied if set before CI RunConfig/PrepareRunConfig step

#### Include tests (required builds will be added automatically):
- [x]  Fast test
- [x]  Integration Tests
- [x]  Stateless tests
- [x]  Stateful tests
- [x]  Unit tests
- [x]  Performance tests
- [x]  All with ASAN
- [x]  All with TSAN
- [x]  All with Analyzer
- [x]  All with Azure
- [ ]  Add your option here

#### Exclude tests:
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with MSAN
- [ ]  All with UBSAN
- [ ]  All with Coverage
- [ ]  All with Aarch64
- [ ]  Add your option here

#### Extra options:
- [ ]  do not test (only style check)
- [ ]  disable merge-commit (no merge from master before tests)
- [ ]  disable CI cache (job reuse)

#### Only specified batches in multi-batch jobs:
- [ ]  1
- [ ]  2
- [ ]  3
- [ ]  4

<details>

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit d6690f8384235531947a09cd65f00623095f3ed4 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/63394/d6690f8384235531947a09cd65f00623095f3ed4/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/interfaces/formats.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ SELECT * FROM nestedt FORMAT TSV
 Differs from `TabSeparated` format in that the rows are written without escaping.
 When parsing with this format, tabs or linefeeds are not allowed in each field.
 
-This format is also available under the name `TSVRaw`.
+This format is also available under the name `TSVRaw`, `Raw`.
```
Comment (yariks5s) [inline docs/en/interfaces/formats.md:209]:
```suggestion
This format is also available under the names `TSVRaw`, `Raw`.
```

File: docs/en/interfaces/formats.md [hunk 2]
```diff
@@ -241,14 +241,14 @@ This format is also available under the name `TSVWithNamesAndTypes`.
 Differs from `TabSeparatedWithNames` format in that the rows are written without escaping.
 When parsing with this format, tabs or linefeeds are not allowed in each field.
 
-This format is also available under the name `TSVRawWithNames`.
+This format is also available under the name `TSVRawWithNames`, `RawWithNames`.
```
Comment (yariks5s) [inline docs/en/interfaces/formats.md:244]:
```suggestion
This format is also available under the names `TSVRawWithNames`, `RawWithNames`.
```

File: docs/en/interfaces/formats.md [hunk 3]
```diff
@@ -241,14 +241,14 @@ This format is also available under the name `TSVWithNamesAndTypes`.
...
 
 ## TabSeparatedRawWithNamesAndTypes {#tabseparatedrawwithnamesandtypes}
 
 Differs from `TabSeparatedWithNamesAndTypes` format in that the rows are written without escaping.
 When parsing with this format, tabs or linefeeds are not allowed in each field.
 
-This format is also available under the name `TSVRawWithNamesAndNames`.
+This format is also available under the name `TSVRawWithNamesAndNames`, `RawWithNamesAndNames`.
```
Comment (yariks5s) [inline docs/en/interfaces/formats.md:251]:
```suggestion
This format is also available under the names `TSVRawWithNamesAndNames`, `RawWithNamesAndNames`.
```

File: tests/queries/0_stateless/00397_tsv_format_synonym.sql [hunk 4]
```diff
@@ -9,3 +9,4 @@ SELECT arrayJoin([1, 2, 3]) AS arr, 'hello' AS s1, 'world' AS s2 FORMAT TSVWithN
 
 SELECT arrayJoin([1, 2, 3]) AS arr, 'hello' AS s1, 'world' AS s2 FORMAT TabSeparatedRaw;
 SELECT arrayJoin([1, 2, 3]) AS arr, 'hello' AS s1, 'world' AS s2 FORMAT TSVRaw;
+SELECT arrayJoin([1, 2, 3]) AS arr, 'hello' AS s1, 'world' AS s2 FORMAT Raw;
```
Comment (yariks5s) [inline tests/queries/0_stateless/00397_tsv_format_synonym.sql:12]: Please add a tests with `RawWithNames` and `RawWithNamesAndNames`

Comment (Unalian) [reply tests/queries/0_stateless/00397_tsv_format_synonym.sql:12]: > Please add a tests with `RawWithNames` and `RawWithNamesAndNames`

Got it!

File: CHANGELOG.md [hunk 5]
```diff
@@ -33,6 +33,7 @@
 * A mode for `topK`/`topkWeighed` support mode, which return count of values and its error. [#54508](https://github.com/ClickHouse/ClickHouse/pull/54508) ([UnamedRus](https://github.com/UnamedRus)).
 * Added function `toMillisecond` which returns the millisecond component for values of type`DateTime` or `DateTime64`. [#60281](https://github.com/ClickHouse/ClickHouse/pull/60281) ([Shaun Struwig](https://github.com/Blargian)).
 * Allow configuring HTTP redirect handlers for clickhouse-server. For example, you can make `/` redirect to the Play UI. [#60390](https://github.com/ClickHouse/ClickHouse/pull/60390) ([Alexey Milovidov](https://github.com/alexey-milovidov)).
+* Allow Raw as a synonym for TSVRaw. [#63394](https://github.com/ClickHouse/ClickHouse/pull/63394) ([Unalian](https://github.com/Unalian))
```
Comment (yariks5s) [inline CHANGELOG.md:36]:
```suggestion

```

You don't need to do it, everything is parsed to changelog automatically🙂

**Review by yariks5s — APPROVED** [review]

**Review by jsc0218 — APPROVED** [review]

**Comment by Blargian** [comment]
Closes #62079

**Pull request merged**